src/lib/style/abstracts/_variables.scss

```diff
@@ -51,37 +73,28 @@ $primary-action-color-active: adjust-color(
   $lightness: 0%
 ) !default;
 
-$primary-action-color-900: adjust-color(
-  $primary-action-color,
-  $saturation: 24%,
-  $lightness: -18%
-) !default;
-
-$primary-action-color-700: adjust-color(
-  $primary-action-color,
-  $saturation: 11%,
-  $lightness: -8%
-) !default;
-
-$primary-action-color-500: adjust-color(
-  $primary-action-color,
-  $saturation: 29%,
-  $lightness: -4%
-) !default;
-
-$primary-action-color-300: adjust-color(
-  $primary-action-color,
-  $saturation: -3%,
-  $lightness: 10%
+$primary-color-palette: (
+  100: adjust-color($primary-action-color, $saturation: -25%, $lightness: 50%),
+  200: adjust-color($primary-action-color, $saturation: -13%, $lightness: 20%),
+  300: adjust-color($primary-action-color, $saturation: -3%, $lightness: 10%),
+  400: adjust-color($primary-action-color, $saturation: -1%, $lightness: 5%),
+  500: $primary-action-color,
+  600: adjust-color($primary-action-color, $saturation: 5%, $lightness: -4%),
+  700: adjust-color($primary-action-color, $saturation: 11%, $lightness: -8%),
+  800: adjust-color($primary-action-color, $saturation: 18%, $lightness: -10%),
+  900: adjust-color($primary-action-color, $saturation: 24%, $lightness: -18%),
```

Comment:
As before, we impute colors if a color palette isn't supplied. This will be useful for portals.

Reply:
The 500 value here has changed; before it was $primary-action-color +saturation -lightness. Now we just use the $primary-action-color.

Some icons/buttons may appear slightly dimmer. I don't think it's a huge deal, but I can check with design on this if it's a bigger issue than I think it is.